lib/ansible/plugins/connection/ssh.py

```diff
@@ -253,6 +253,8 @@ def _send_initial_data(self, fh, in_data):
 
         try:
             fh.write(in_data)
+            if tty:
+                fh.write("__EOF__\n")
```

Comment:
AFAIK it's not valid Python, so I'd think zero? But if you have a better suggestion, I'm happy to change it.